### PR TITLE
fix:(#332) : update theme.js

### PR DESCRIPTION
**Pull request by sanoojes**
Solution from https://github.com/nimsandu/spicetify-bloom/pull/337 Commented it out

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -450,17 +450,17 @@
...
+//     waitForElements(
+//       ['.main-topBar-container', '.Root__main-view'],
+//       ([topBarContainer, rootMainView]) => {
+//         rootMainView.insertAdjacentElement(
+//           to === 'up' ? 'beforebegin' : 'afterbegin',
+//           topBarContainer
+//         );
+//       }
+//     );
+//   }
```
Comment (kernelb00t) [inline src/theme.js:463]: Why not remove it entirely?

Comment (sanoojes) [reply src/theme.js:463]: yeah i will do it

Comment (nimsandu) [reply src/theme.js:463]: @kernelb00t thanks for helping us out!!

@sanoojes lemme knw once you are done with it!

Comment (sanoojes) [reply src/theme.js:463]: okee

**Pull request closed without merging**